.github/workflows/tests.yml

```diff
@@ -14,8 +14,13 @@ jobs:
       matrix:
         os: [ubuntu-latest, macOS-latest]
     steps:
+    - name: Check out Git repository
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
```

Comment:
Splitting this into two steps with a `fetch-depth: 0` makes this slower rather than faster (see other CI jobs).